homework-submission/react-hyf/client/src/components/week3-homework/week3.css

```diff
@@ -0,0 +1,20 @@
+.hidden {
+  display: none;
+}
+
+.user_data_wrapper {
+  background: #8fc06c;
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  top: 0px;
+  left: 0;
+  z-index: 999;
+  display: block;
+}
+
+.header {
+  position: absolute;
+  left: 36rem;
```

Comment:
This can't be the best way to center an element, here is some reading material; https://css-tricks.com/almanac/properties/j/justify-content/